src/OnboardingSPA/components/Header/components/SiteGenHeader/step-navigation.js

```diff
@@ -51,20 +50,13 @@ const StepNavigation = () => {
 	}, [] );
 
 	const isFirstStep = null === previousStep || false === previousStep;
-	const isPreviewStep = currentStep?.path === stepSiteGenPreview?.path;
-
+	const isEditorStep = currentStep?.path === stepSiteGenEditor?.path;
 	return (
 		<div className="nfd-onboarding-header--sitegen__step-navigation">
-			{ isFirstStep ? (
+			{ isFirstStep || isEditorStep ? (
 				<></>
 			) : (
-				<Back
-					path={
-						isPreviewStep
-							? stepSiteGenSiteLogo.path
-							: previousStep.path
-					}
-				/>
```

Comment:
I think this is quite crucial as why we included this was when a user does a Browser back on Preview page he ends up on the same page again as experience redirects him back to this page and he can never go back

Reply:
Since we are removing the navigation bar itself from the preview step. the check for the isPreviewStep is not relevant anymore on the back button. The behaviour to stay redirect back on the preview step is still the same.

Reply:
Let's confirm if it's OK to hide the header for that step.

Reply:
@officiallygod, @ajayadav09 has confirmed that we have a go ahead.